<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>346</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>871</width>
      <height>411</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_2{
	background-color: qlineargradient(spread:pad, x1:0.097, y1:0.448864, x2:0.631, y2:0.409091, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));

	color:rgba(255,255,255,210);
    border-radius:5px;
}
QPushButton#pushButton_2:hover{
	background-color: qlineargradient(spread:pad, x1:0.097, y1:0.448864, x2:0.631, y2:0.409091, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}
QPushButton#pushButton_2:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(150,123,111,255);
}




QPushButton#pushButton_3{
	background-color:rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}
QPushButton#pushButton_3:hover{
	color:rgba(131,96,53,255);
}
QPushButton#pushButton_3:pressed{
	padding-left:5px;
	padding-top:5px;
	color:rgba(91,88,53,255);
}</string>
     </property>
     <property name="text">
      <string>Add Contact</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>421</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>240</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Contact</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>64</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Admin Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton{
	background-color: qlineargradient(spread:pad, x1:0.097, y1:0.448864, x2:0.631, y2:0.409091, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));

	color:rgba(255,255,255,210);
    border-radius:5px;
}
QPushButton#pushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0.097, y1:0.448864, x2:0.631, y2:0.409091, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}
QPushButton#pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(150,123,111,255);
}




QPushButton#pushButton_1{
	background-color:rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}
QPushButton#pushButton_1:hover{
	color:rgba(131,96,53,255);
}
QPushButton#pushButton_1:pressed{
	padding-left:5px;
	padding-top:5px;
	color:rgba(91,88,53,255);
}</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>202</x>
       <y>60</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>300</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ClosedHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_12{
	background-color: qlineargradient(spread:pad, x1:0.097, y1:0.448864, x2:0.631, y2:0.409091, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));

	color:rgba(255,255,255,210);
    border-radius:5px;
}
QPushButton#pushButton_12:hover{
	background-color: qlineargradient(spread:pad, x1:0.097, y1:0.448864, x2:0.631, y2:0.409091, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}
QPushButton#pushButton_12:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(150,123,111,255);
}




QPushButton#pushButton_13{
	background-color:rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}
QPushButton#pushButton_13:hover{
	color:rgba(131,96,53,255);
}
QPushButton#pushButton_13:pressed{
	padding-left:5px;
	padding-top:5px;
	color:rgba(91,88,53,255);
}</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
